Array/script.js

```diff
@@ -0,0 +1,28 @@
+// Create an Array #1
+var numbers = new Array()
+numbers[0] = 1
+numbers[1] = 2
+numbers[2] = 3
+numbers[3] = 4
+numbers[4] = 5
+
+// Create an Array #2
+var names = ["kaveh","kiarash"]
+
+// push it to the end of the Array
+names.push("mohammad")
+
+// Delete the last array index
+names.pop()
+
+// Delete an array using index
+delete names[0]
+
+// find an array
+names.find(name => name == "mohammad") // return Mohammad
```

Comment:
how would you use this code in an application? whats the return value of find? true false? and array?

Reply:
find return value

Reply:
If I wanna use this code in my app, I can use it in if statement and for loop and place input ( textbox ) instead of "mohammad".

Reply:
code here!!